src/app/tasks/components/TaskLoggerItem/TaskLoggerItem.tsx

```diff
@@ -72,6 +72,7 @@ const TaskLoggerItem = ({ notification, task }: TaskLoggerItemProps): JSX.Elemen
   const [isHovered, setIsHovered] = useState(false);
   const [isRetryActionDisabled, setIsRetryActionDisabled] = useState(false);
   const selectedWorkspace = useSelector(workspacesSelectors.getSelectedWorkspace);
+  const [nItems, setNItems] = useState(0);
```

Comment:
why not pass directly `nItems` to `TaskLoggerActions`?

Reply:
Your are right. Changed!